docs/reference/ccr/remote-recovery.asciidoc

```diff
@@ -0,0 +1,67 @@
+[role="xpack"]
+[testenv="platinum"]
+[[remote-recovery]]
+== Remote recovery
+
+Remote recovery is the process used to build a new copy of a shard on a follower
+node by copying data from the primary shard in the leader cluster. {es} uses this
+remote recovery process to bootstrap a follower index using the data from the
+leader index. This allows the follower to receive a copy of the current state of
+the leader index, even if a complete history of changes is not available on the
+leader due to Lucene segment merging.
+
+Remote recovery is a network intensive process that transfers all of the Lucene
+segment files from the leader cluster to the follower cluster. The follower
+requests that a recovery session be initiated on the primary shard in the leader
+cluster. The follower then requests file chunks concurrently from the leader. By
+default, the the process concurrently requests `5` large `1mb` file chunks as remote
+recovery is designed to support leader and follower clusters with high network
+latency between them.
+
+Information about an in-progress remote recovery can be obtained using the
+{ref}/cat-recovery.html[recovery API] on the follower cluster. Remote recoveries are implemented
+using the {ref}/modules-snapshots.html[snapshot and restore] infrastructure. This means that
+on-going remote recoveries will be labelled as type `snapshot` in the recovery api.
```

Comment:
```suggestion
on-going remote recoveries are labelled as type `snapshot` in the recovery API.
```

Reply:
Is there some way to know when the recovery process is complete (other than using the recovery API)?

Reply:
The recovery API is the primary. You could probably use the `cat indices` api to check if the index is green. You can also add a parameter to a follower request to wait until the process is completed. However, that is documented on the put follow request page.